imageGenerator/imageGen.py

```diff
@@ -0,0 +1,83 @@
+import json
+import os
+import sys
+
+import requests
+from dotenv import load_dotenv
+from requests.exceptions import RequestException
+
+# stable-diffusion-xl image gen-------------------------------------------------
+
+from diffusers import DiffusionPipeline
+from PIL import Image 
+import torch
+
+pipe = DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16, use_safetensors=True, variant="fp16")
+pipe.to("cuda")
+
+# --------------------------------------------------------------------
+
+dot = os.path.dirname(os.path.realpath(__file__))
+load_dotenv(f"{dot}/../local.env") # todo: refactor this to check for containerized builds, in which case we should use the `virtual.env` file
+API_PORT = os.getenv("API_PORT")
+API_HOST = os.getenv("API_HOST")
+
+recordImageURL = f'http://{API_HOST}:{API_PORT}/article.recordImage'
+getAllImageURL = f'http://{API_HOST}:{API_PORT}/article.getAllImageGen'
+
+def record_image(id):
+
+    try: 
+        response = requests.put(
```

Comment:
@gpseal Might want to add `TOKEN` to the `headers` before accessing the route. Just add `headers` parameter in your parenthesis:
```python
headers={'Content-Type': 'application/json', 'Authorization': f'{TOKEN}'}
```

Reply:
@ArtemKech see above comment.